src/app/components/SettingsTab.js

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
Please add comment inside this Component class that this component is only visual placeholder